tap_chargebee/schemas/common/payment_sources.json

```
@@ -75,6 +78,73 @@
         }
```

```suggestion
        },
        "email": {
          "type": ["null", "string"]
        }
```

Chargebee documentation implies this field also exists: https://apidocs.eu.chargebee.com/docs/api/payment_sources?lang=curl#payment_source_bank_account_email

I was going to make a pull request to cover this change, but I think it makes more sense as an addition to this one if possible